build/ci/.azure-devops-wasm-uitests.yml

```diff
@@ -83,7 +91,7 @@ jobs:
 
   - task: DownloadBuildArtifacts@0
     inputs:
-        artifactName: wasm-uitest-binaries
+        artifactName: wasm-uitest-binaries-UWP
```

Comment:
Note: Few lines below we run `dotnet-serve` on the `site-net7.0-UWP` directory. So we previously were downloading both UWP and WinUI artifacts and then using only UWP artifacts. The separation allows for granular download, which should be faster.